core/tests/test_icons.py

```diff
@@ -127,6 +134,19 @@ def test_create_fallback(app, capsys):
     )
 
 
+def test_create_fallback_unloadable(monkeypatch, app, capsys):
+    """If a resource exists, but can't be loaded, an error is raised."""
```

Comment:
This should be documented. Right now the Icon documentation doesn't mention the possibility of raising any exceptions at all.